### ISSUE TITLE
[ Windows ] AWTGLCanvas gives error when activating sRGB

### ISSUE THREAD

**Issue by JNightRider** [LWJGLX/lwjgl3-awt#78]
Hello everyone.

I recently had an issue with Windows 11 turning on the `sRGB` option. If I run lwjgl3-awt on Linux, it runs without problems ~~(same computer with dual boot)~~.

[ Error - Output ]
```shell
Exception in thread "AWT-EventQueue-0" java.lang.RuntimeException: Exception while creating the OpenGL context
	at org.lwjgl.opengl.awt.AWTGLCanvas.beforeRender(AWTGLCanvas.java:81)
	at org.lwjgl.opengl.awt.AWTGLCanvas.render(AWTGLCanvas.java:120)
	at org.lwjgl.opengl.awt.AWTTest$2.run(AWTTest.java:60)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.awt.AWTException: sRGB color space requested but WGL_EXT_framebuffer_sRGB is unavailable
	at org.lwjgl.opengl.awt.PlatformWin32GLCanvas.create(PlatformWin32GLCanvas.java:435)
	at org.lwjgl.opengl.awt.PlatformWin32GLCanvas.create(PlatformWin32GLCanvas.java:155)
	at org.lwjgl.opengl.awt.AWTGLCanvas.beforeRender(AWTGLCanvas.java:79)
	... 15 more
```

This can be reproduced using the examples provided by lwjgl3-awt by turning on the 'sRGB' option

[ **Device Information** ]

- OS: WIndows 11
- Graphic card: GTX 1650

**Comment by JNightRider** [comment]
[PR](https://github.com/LWJGLX/lwjgl3-awt/pull/81) that solve this problem.

**Comment by SWinxy** [comment]
Fixed in 7043714eb6bdc82f94117a2aa3a6dfb30e57fd20
